architecture/core_concepts/routes.adoc

```diff
@@ -171,34 +171,9 @@ no-route-hostname.my-namespace.router.default.svc.cluster.local <1>
 *router.default.svc.cluster.local*.
```

Comment:
Can I just point out that the asterisks above in `*$routename[.$namespace].$suffix*` come out literally in the build? Pre-existing problem...